src/resources/orders/orderlines/OrderLinesResource.ts

```diff
@@ -87,6 +87,6 @@ export default class OrdersLinesResource extends ParentedResource<OrderData, Ord
       throw new ApiError('The order id is invalid');
     }
     const { orderId: _, ...data } = parameters;
-    return this.network.delete(this.getResourceUrl(orderId)) as Promise<true>;
+    return this.network.delete<true>(this.getResourceUrl(orderId));
```

Comment:
What's the reason the `OrdersResource` passes in an `Order` type, while this one passes in `true`?

Reply:
Because some endpoints return 200 along with the model, while others return 204.

See [cancel order](https://docs.mollie.com/reference/v2/orders-api/cancel-order#response) vs [cancel order lines](https://docs.mollie.com/reference/v2/orders-api/cancel-order-lines#response).

Previously, the delete-y methods returned `Promise<Model | boolean>` according to the type definition. Now we specify which of those ‒ the model or `true` ‒ is returned in that definition.

Reply:
We could introduce an enum for this if we feel that is helpful in the future.